aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -171,10 +174,11 @@ Module& Module::_impl(const char* name_str, CppFunction&& f) & {
   return *this;
 }
 
-Module& Module::_fallback(CppFunction&& f) & {
-  TORCH_CHECK(!ns_, "Cannot define fallbacks from namespaces, use c10::import().fallback() instead");
-  TORCH_CHECK(f.dispatch_key_, "Fallback for catch all function not supported");
-  registrars_.emplace_back(Dispatcher::singleton().registerFallback(*f.dispatch_key_, std::move(f.func_)));
+Library& Library::_fallback(CppFunction&& f) & {
+  TORCH_CHECK(!ns_, "Cannot define a fallback in TORCH_LIBRARY (it would affect operators outside of your library); did you mean to use TORCH_LIBRARY_IMPL instead");
```

Comment:
I found the "did you mean" part kind of confusing - it sounded like maybe there was a scoped version of fallback you could somehow specify using TORCH_LIBRARY_IMPL. 

I'm not sure how to rephrase it to more clearly connote "maybe you don't reallllly want a fallback", but absent a rephrasing maybe it's best to leave it off entirely?